truss/remote/baseten/remote.py

```diff
@@ -70,6 +71,34 @@ def push(  # type: ignore
             truss_handle=truss_handle,
         )
 
+    def get_baseten_service(
+        self, model_name: str, published: bool = False
+    ) -> BasetenService:
+        model_id, model_versions = get_model_versions_info(self._api, model_name)
+        model_version = None
+        if published:
+            for mv in model_versions:
+                if not mv["is_draft"]:
+                    model_version = mv
```

Comment:
what assumptions are we making about order here? what if there are multiple non-draft versions that get returned

Reply:
yeah. this needs to modify the query to sort by return time. I'll play around with the query a bit to get this right. I'd rather have a good graphql query that returns the right version instead of doing the post processing here.